crates/executors/src/executors/opencode.rs

```diff
@@ -127,3 +155,15 @@ impl StandardCodingAgentExecutor for Opencode {
         }
     }
 }
+
+fn default_to_true() -> bool {
+    true
+}
+
+fn setup_approvals_env(auto_approve: bool, env: &ExecutionEnv) -> ExecutionEnv {
```

Comment:
Should we configure all of these permissions settings? https://opencode.ai/docs/permissions/

We should be asking for permission before executing web searches or running commands outside of the current directory. I'm not sure what `doom_loop` is.